src/Symfony/Bundle/FrameworkBundle/Secret/FilesSecretStorage.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Secret;
+
+use Symfony\Bundle\FrameworkBundle\Exception\SecretNotFoundException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author Tobias Schultze <http://tobion.de>
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class FilesSecretStorage implements MutableSecretStorageInterface
+{
+    private const FILE_SUFFIX = '.bin';
+
+    private $secretsFolder;
+    private $encoder;
+    private $filesystem;
+
+    public function __construct(string $secretsFolder, EncoderInterface $encoder)
+    {
+        $this->secretsFolder = rtrim($secretsFolder, \DIRECTORY_SEPARATOR);
```

Comment:
```suggestion
        $this->secretsFolder = rtrim($secretsFolder, '\\/');
```

One can pass `/` while the operation is on Windows.